src/main/webapp/resources/js/pages/cart/components/share/index.js

```diff
@@ -0,0 +1,10 @@
+import { Row } from "antd";
+import React from "react";
+
```

Comment:
Add a file comment for this

Reply:
Added in 30d93f2b20e775f8f08280946cda54c892eb7d20